tests/pds/peppi/quickstart.py

```diff
@@ -12,7 +12,7 @@
 # in the `PDS keyword search <https://pds.nasa.gov/datasearch/keyword-search/search.jsp>`_
 mercury_id = "urn:nasa:pds:context:target:planet.mercury"
 # filter here:
-products = pep.Products(client).has_target(mercury_id).before(date1).observationals()
+products = pep.Products(client).has_target("Mercury").before(date1).observationals()
```

Comment:
Since `mercury_id` is still defined here, maybe we should include an example of both variations of the new `has_target` convention here.

Reply:
Correct, I will fix that.

Reply:
Actually I removed it for now since the plan is to have only the keyword search in the quickstart. As we need to enrich the quickstart we can had back new cases as they come in the quickstart in the online doc.